backend/.env.example

```
@@ -1,6 +1,7 @@
 SAFEQUERY_APP_NAME=SafeQuery API
 SAFEQUERY_APP_POSTGRES_URL=postgresql://safequery:safequery@127.0.0.1:5432/safequery
 SAFEQUERY_ENVIRONMENT=development
+SAFEQUERY_DEV_AUTH_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SAFEQUERY_DEV_AUTH_ENABLED key should go before the SAFEQUERY_ENVIRONMENT key

(UnorderedKey)